cilium/rockcraft.yaml

```diff
@@ -15,7 +15,7 @@ environment:
 
 services:
   cilium:
-    command: /usr/bin/cilium
+    command: /usr/bin/cilium-dbg
```

Comment:
what's the `-dbg` command?  is this an intended switch

Reply:
The `cilium-dbg` command is just a rename of the `cilium` binary that's in agent images(cli), so the switch is intentional. For reference you can check [upstream image](https://github.com/cilium/cilium/blob/7cf5782920423b07e7a4e6e67fafd81e1d4a1d33/images/cilium/Dockerfile#L94)